httpapi/httpapi.go

```diff
@@ -1,6 +1,7 @@
 package httpapi
```

Comment:
You'll see many methods changing receiving a `context.Context` now because part of what changed in`go-datastore` dependency is now receiving a `ctx` for `Get`, `Put`, and others (mostly, all of them).

I'll point out in a comment when the bubble starts, and then "goes up" to our methods from there.